admin/installer/_images.py

```diff
@@ -3,7 +3,60 @@
 """
 Helper utilities for CloudFormation Installer's Packer images.
 """
-from pyrsistent import freeze
+import json
+import sys
+from tempfile import mkdtemp
+
+import boto3
+
+from effect import (
+    Effect, ComposedDispatcher, TypeDispatcher,
+    sync_performer, base_dispatcher,
+)
+from effect.do import do, do_return
+
+from txeffect import deferred_performer, perform as async_perform
+
+from pyrsistent import PClass, field, freeze, thaw, pvector_field
+
+from twisted.python.filepath import FilePath
+from twisted.python.usage import Options, UsageError
+
+from flocker.common.runner import run
+
+
+PACKER_TEMPLATE_DIR = FilePath(__file__).sibling('packer')
+
+# The AWS regions supported by Packer.
+# XXX ap-northeast-2 is not supported by the current packer release:
+# https://github.com/mitchellh/packer/issues/3058
+AWS_REGIONS = (
+    u"ap-northeast-1",
+    u"ap-southeast-1",
+    u"ap-southeast-2",
+    u"eu-central-1",
+    u"eu-west-1",
+    u"sa-east-1",
+    u"us-east-1",
+    u"us-west-1",
+    u"us-west-2",
+)
+# Hard coded Ubuntu 14.04 base AMI.
+# XXX These AMIs are constantly being updated.
+# This should be looked up dynamically using Canonical's AWS owner ID
+# 099720109477. See:
+# http://askubuntu.com/a/53568
```

Comment:
This sent me to something about ext on windows. I think you may have meant http://askubuntu.com/questions/53582